### DISCUSSION TITLE
Errore sulla generazione dei file pdf

### DISCUSSION THREAD

**Discussion by istitutoenricomattei** [iisgiua/giuaschool#746] [Richieste di aiuto]
Buonasera,
ho riscontrato un'errore dopo essere passato dalla versione 1.5.3 alla 1.5.9 reinstallando tutto. Siccome necessitavo di reinserire tutti i docenti necessitavo anche di averne le password per inoltrarle, ma quando vado a cliccare "scarica password" di qualsiasi utente io crei mi porta ad una pagina di errore. Come posso risolvere?

**Comment by registro-giuaschool** [comment]
Salve può contattarmi dalla seguente e-mail:
registroelettronicogiuaschool@gmail.com

**Reply by trinko** [reply]
> Salve può contattarmi dalla seguente e-mail: [registroelettronicogiuaschool@gmail.com](mailto:registroelettronicogiuaschool@gmail.com)

Se intendi aiutare l'utente va benissimo, ma è meglio che dichiari in modo esplicito se il tuo intervento è gratuito o no. Un messaggio come questo sembra ambiguo.

_In ogni caso, raccomando agli utenti di non condividere le credenziali del proprio registro elettronico con nessuno, a meno che non siate sicuri della sua identità._

**Reply by registro-giuaschool** [reply]
Salve, non sono qui per essere pagato o rubare credenziali. Sono qui per aiutare! 😊

**Comment by trinko** [comment]
> Buonasera, ho riscontrato un'errore dopo essere passato dalla versione 1.5.3 alla 1.5.9 reinstallando tutto. Siccome necessitavo di reinserire tutti i docenti necessitavo anche di averne le password per inoltrarle, ma quando vado a cliccare "scarica password" di qualsiasi utente io crei mi porta ad una pagina di errore. Come posso risolvere?

Per prima cosa: il passaggio di versione l'hai fatto tramite lo strumento di aggiornamento dell'apposito menu? In altre parole, sei passato dalla 1.5.3 alla 1.5.4, poi alla 1.5.5 ecc.?
In caso contrario è normale che riscontri problemi.

Comunque, apri una segnalazione di errore e riporta il log come indicato qui: [Come fare una segnalazione di errore](https://github.com/iisgiua/giuaschool/discussions/576)
